src/ppx_deriving_protobuf.cppo.ml

```diff
@@ -1,8 +1,20 @@
-#if OCAML_VERSION >= (4, 06, 0)
+#if OCAML_VERSION >= (4, 08, 0)
+#define Rtag(label, attrs, has_empty, args) \
+        Rtag({ txt = label }, has_empty, args)
```

Comment:
I think that this definition is wrong, and in particular the fact that `attrs` is completely ignored is the sign that something is amiss. Try:

```ocaml
#define Rtag(label, attrs, has_empty, args) \
{ \
  prf_desc = Rtag({txt = label}, has_empty, args); \
  prf_attributes = attrs; \
}
```

Moreover, I think that it could be nice to define this macro as `Rtag_patt` instead of `Rtag`, to follow the new convention of `_patt` and `_expr` macros.

Reply:
(It would be `Rtag_patt` for all OCaml versions, not just >=4.08)